geoips/commandline/create_sector_image.py

```diff
@@ -3,6 +3,9 @@
 import os
 import argparse
 from geoips import interfaces
+import logging
+
+LOG = logging.getLogger(__name__)
```

Comment:
@jsolbrig If this is called at the command line, do we need to use "setup_logger" here rather than just logging.getLogger(__name__).  I'm not sure this will work as-is?

Reply:
Yeah, you're right about this one. I just updated and pushed a change for this.